.php-cs-fixer.dist.php

```diff
@@ -41,6 +41,8 @@
 		// So we use target PHP70 for the moment.
 		'@PHP70Migration' => true,
 		//'@PHP71Migration' => true,
+                // Avoid adding public to const (incompatible with PHP 7.0):
+                'visibility_required' => ['elements'=>['property', 'method']],
```

Comment:
This rule is not the only one to not execute.
The only solution to be sure to not execute rule that are not "minimum version" compliant is to install php-cs-fixer by using the good php version.  If you install php-cs-fixer and runit from php8, you will also break compatibility with 7.4
This is why php version is forced into file php-cs-fixer/run-php-cs-fixer.sh

Reply:
"Absolutely sure" - certainly.

However, the configuration also allows to exclude conversions - updating the configuration to not apply this change does not exclude the other approach to help avoid unwanted changes.

The line
```
                'visibility_required' => ['elements'=>['property', 'method']],
```
Overrides the configuration for the PSR12 rule that applies public on consts .

The tool should have included this conversion rule in `@PHP71Migration` - I filed an [issue in php-cs-fixer](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/issues/7526)